magento/pwa-studio
# [Doc] Update changelog for 6.0.1

## Description

Update the changelog for release 6.0.1

## Related Issue

Closes #ISSUE_NUMBER.

## Acceptance 

@dpatil-magento 

### Verification Stakeholders

@dpatil-magento 

### Specification

### Verification Steps

1. Open CHANGELOG file
1. Verify content is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
